lua/lazyvim/plugins/extras/vscode.lua

```diff
@@ -35,6 +35,8 @@ vim.api.nvim_create_autocmd("User", {
     vim.keymap.set("n", "<leader><space>", "<cmd>Find<cr>")
     vim.keymap.set("n", "<leader>/", [[<cmd>lua require('vscode').action('workbench.action.findInFiles')<cr>]])
     vim.keymap.set("n", "<leader>ss", [[<cmd>lua require('vscode').action('workbench.action.gotoSymbol')<cr>]])
+    vim.keymap.set("n", "]d", [[<cmd>lua require('vscode').call('editor.action.marker.next')<cr>]], { desc = "Next Diagnostic" })
```

Comment:
OP here: alternative to this would be to update the `diagnostic_goto()` function in [keymaps.lua](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/config/keymaps.lua#L100-L109).

The change in this PR can go here in `vscode.lua` or `keymaps.lua`, though I think vscode.lua seems the simpler of the two. Let me know :)